cardano-tx-generator/test/script.json

```diff
@@ -84,6 +84,13 @@
         ]
     },
     {
-        "runBenchmark": "txlist"
-    }
+        "asyncBenchmark": "thread1",
+        "txList": "txlist"
+    },
+    {
+        "waitForEra": "Byron"
```

Comment:
Wait for.. Byron?  I think I'm missing something..

What does that mean?

Reply:
Right that does not really make sense.
Though the effect would be wait forever...
This is just the 'testscript' but this just reminds me.. the real script needs to be updated